src/test/java/com/edmunds/rest/databricks/service/ScimServiceTest.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright 2018 Edmunds.com, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.edmunds.rest.databricks.service;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+import com.edmunds.rest.databricks.DTO.scim.group.GroupDTO;
+import com.edmunds.rest.databricks.DTO.scim.user.EntitlementDTO;
+import com.edmunds.rest.databricks.DTO.scim.user.EntitlementsDTO;
+import com.edmunds.rest.databricks.DTO.scim.user.UserDTO;
+import com.edmunds.rest.databricks.DatabricksRestException;
+import com.edmunds.rest.databricks.DatabricksServiceFactory;
+import com.edmunds.rest.databricks.fixtures.DatabricksFixtures;
+import java.io.IOException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class ScimServiceTest {
+
+  public static final String USERNAME = "testintegration@test.com";
+  public static final String FAMILY_NAME = "klaus";
+  public static final String GIVEN_NAME = "iohannis";
+  private ScimService service;
+
+
+  private long idUSer1;
+  private final String ADMIN_GROUP = "admins";
+  private String testGroup1 = "TestIntegration Group";
+  private long idGroup1;
+
+
+  @BeforeClass
+  public void setUpOnce() {
+    DatabricksServiceFactory factory = DatabricksFixtures.getDatabricksServiceFactory();
+    service = factory.getScimService();
+  }
+
+  @AfterClass(alwaysRun = true)
+  public void cleanUp() {
+    deleteAllTestUsers();
+    deleteAllTestGroups();
+  }
+
+  @AfterMethod
+  public void tearDown() {
+    deleteAllTestUsers();
+    deleteAllTestGroups();
+  }
+
+  public void deleteAllTestUsers() {
+    try {
+      if (idUSer1 != 0) {
+        service.deleteUser(idUSer1);
+        idUSer1 = 0;
+      }
+    } catch (Exception e) {
+      //ignore
```

Comment:
Why ignore? Could you at least print it out?

Reply:
+1